catalog/docs/src/main/resources/IntContents.adoc

```diff
@@ -1194,142 +1194,23 @@ Otherwise, caching is canceled if the user cancels the product download.
-This file is referred to as a "Schematron ruleset."
-These rules are contained in one `.sch` file or a hierarchy of `.sch` files.
-However, there is ultimately one `.sch` file that includes or uses all of the other `.sch` files.
-This one `.sch` file is the "ruleset" used by the {branding} Schematron Validation Service.
-
-Schematron also includes SVRL (Schematron Validation Report Language) report generation, which is in XML format.
-This report includes the results of all of the Schematron rulesets' assertions, classifying them as warnings or errors (based on the ruleset).
-
-{branding} implements Schematron as a Pre-Ingest Plugin, running the Schematron ruleset(s) against each catalog entry in each `create` and `update` ingest request that {branding} receives.
-The {branding} Schematron Validation Pre-Ingest Plugin consists of two components: the Schematron "engine" and the client ruleset bundle(s).
-Each are described below.
-
-===== Schematron Validation Plugin
-
-The Schematron Validation Service is in a single OSGi bundle named `plugin-schematron-validation`.
-This bundle includes all of the code to implement:
-
-. Loading and pre-compilation of the client ruleset bundle
-. Executing the ruleset against ingest requests
-. Generating the SVRL report. From this report, the Schematron Validation Service determines if errors and/or warnings were detected during validation. If errors or warnings exist, validation fails and the ingest request is rejected. A SOAP fault is then returned to the client, including details on why the request is invalid.
-
-The client's ruleset bundle determines what rules generate warnings and what rules generate errors.
-The Schematron Validation Service provides a configuration option (accessible via the Admin Console's Configuration page) to suppress warnings.
-When this option is set, if only warnings are detected during Schematron validation, then the request is considered valid.
-By default, this suppress warnings option is unset (hence warnings result in invalid requests by default).
-
-Validation is executed per catalog entry in the ingest request.
-If multiple catalog entries are in the request, Schematron validation stops once a catalog entry is determined to be invalid.
-For example, if ten catalog entries are in a singe create ingest request and entry #4 is invalid, entries 5 through 10 will not even be validated. Schematron returns an invalid status after entry #4 is validated.
+Custom schematron rulesets can be used to validate metacard metadata.
+Multiple services can be created, and each service can have multiple rule sets associated with it.
+Namespaces are used to distinguish services.
+The root schematron files may be placed anywhere on the file system as long as they are configured with an absolute path.
+Any root schematron files with a relative path is assumed to be relative to DDF_HOME/schematron.
```

Comment:
I was afraid of that. I have some ideas on possible workarounds, but that's a different topic.